0001-Rename-app-to-net.base_art.Glide.patch

```diff
@@ -0,0 +1,288 @@
+From 2d2eb66a2e8a6d6116d959cc58bba152e8bb6160 Mon Sep 17 00:00:00 2001
+From: Philippe Normand <phil@base-art.net>
+Date: Tue, 20 Aug 2024 09:32:22 +0100
+Subject: [PATCH] Rename app to net.base_art.Glide
+
+See #125
+---
+ .github/workflows/flatpak.yml                  |  2 +-
+ README.md                                      |  8 ++++----
+ ...evel.json => net.base_art.Glide.Devel.json} | 18 +++++++++---------
+ ...lic.svg => net.base_art.Glide-symbolic.svg} |  0
+ ....Devel.svg => net.base_art.Glide.Devel.svg} |  0
+ ...ource.svg => net.base_art.Glide.Source.svg} |  2 +-
+ ...lide.desktop => net.base_art.Glide.desktop} |  2 +-
+ ...nfo.xml => net.base_art.Glide.metainfo.xml} | 12 +++++++++---
+ ...aseart.Glide.svg => net.base_art.Glide.svg} |  0
+ ....baseart.Glide.ui => net.base_art.Glide.ui} |  0
+ glide.flatpakref                               |  4 ++--
+ justfile                                       |  6 +++---
+ meson.build                                    | 10 +++++-----
+ src/ui_context.rs                              |  6 +++---
+ 14 files changed, 38 insertions(+), 32 deletions(-)
+ rename build-aux/{net.baseart.Glide.Devel.json => net.base_art.Glide.Devel.json} (84%)
+ rename data/{net.baseart.Glide-symbolic.svg => net.base_art.Glide-symbolic.svg} (100%)
+ rename data/{net.baseart.Glide.Devel.svg => net.base_art.Glide.Devel.svg} (100%)
+ rename data/{net.baseart.Glide.Source.svg => net.base_art.Glide.Source.svg} (99%)
+ rename data/{net.baseart.Glide.desktop => net.base_art.Glide.desktop} (99%)
+ rename data/{net.baseart.Glide.metainfo.xml => net.base_art.Glide.metainfo.xml} (95%)
+ rename data/{net.baseart.Glide.svg => net.base_art.Glide.svg} (100%)
+ rename data/{net.baseart.Glide.ui => net.base_art.Glide.ui} (100%)
```

Comment:
this patchfile seems to have a lot of unrelated and unnecessary changes.

You have a `flatpakref` in there.

Icons metainfo and desktop file can be renamed using `rename-icon, rename-desktop-file, rename-appdata-file`. see https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html

The rest should be trimmed a bit, for example, we don't need changes in the manifest in git or your github workflows.

Reply:
Eventually I need the Devel flatpak renamed too, for consistency. So I'm not sure what's the point of splitting these changes away.

Reply:
> Icons metainfo and desktop file can be renamed using rename-icon, rename-desktop-file, rename-appdata-file. see https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html

Yes sure, then I would end-up with a mix of files, some keeping the old name and some the new one? I'm not sure that's better :(

Reply:
Alright, I can do that ;) I'll update this PR for 0.6.5 then.